WordPress/Classes/ViewRelated/Post/Revisions/RevisionsTableViewController.swift

```diff
@@ -76,8 +81,40 @@ private extension RevisionsTableViewController {
     }
 
     @objc private func refreshRevisions() {
+        if sectionCount == 0 {
+            configureAndDisplayNoResults(title: NoResultsText.loadingTitle,
+                                         accessoryView: NoResultsViewController.loadingAccessoryView())
+        }
+
         manager?.getRevisions()
     }
+
+    private func configureAndDisplayNoResults(title: String,
+                                      subtitle: String? = nil,
+                                      buttonTitle: String? = nil,
+                                      accessoryView: UIView? = nil,
+                                      forNoSearchResults: Bool = false) {
```

Comment:
It appears `subtitle` and `forNoSearchResults` are not used. Can they be removed?

Reply:
You are right! I removed `forNoSearchResults`. Subtitle now is used when a no result is displayed.